tests/unit/analytics-test.js

```diff
@@ -34,9 +34,8 @@ describe('analytics', function() {
     command = null;
   });
 
-  it('track gets invoked on command.validateAndRun()', function() {
-    return command.validateAndRun([]).then(function() {
-      expect(called, 'expected analytics.track to be called').to.be.true;
-    });
+  it('track gets invoked on command.validateAndRun()', async function() {
+    await command.validateAndRun([]);
+    return expect(called, 'expected analytics.track to be called').to.be.true;
```

Comment:
the `return` here is no longer necessary. we only needed to return in the previous state because of the Promise, but since the async function already implicitly returns a Promise the explicit return is obsolete.

Reply:
I know, I was debating it if I should add it, I wasn't sure if we want to return a Promise resolving to undefined. I feel this might be more explicit (although admittedly more noisy too).

I don't have strong opinions on this question, so I'm happy to remove it if you want.